projects/storefrontlib/src/lib/auth/services/client-error/client-error-handling.service.ts

```diff
@@ -26,17 +22,8 @@ export class ClientErrorHandlingService {
     );
   }
 
-  private loadNewClientToken(): Observable<any> {
-    return this.store.pipe(
-      select(fromStore.getClientTokenState),
-      tap((state: ClientTokenState) => {
-        if (!state.loading) {
-          this.store.dispatch(new fromStore.LoadClientToken());
-        }
-      }),
-      filter((state: ClientTokenState) => state.loaded),
-      map((state: ClientTokenState) => state.token)
-    );
+  private loadNewClientToken(): Observable<ClientAuthenticationToken> {
```

Comment:
we don't need this one line function, right? We can just call this.authService.refreshClientToken() in handleExpiredClientToken.

Reply:
Yes, good catch